extensions/shared/register-jetpack-block.js

```diff
@@ -12,9 +13,21 @@ import getJetpackExtensionAvailability from './get-jetpack-extension-availabilit
 import withHasWarningIsInteractiveClassNames from './with-has-warning-is-interactive-class-names';
 import wrapPaidBlock from './wrap-paid-block';
 
+const availableBlockTags = {
+	paid: __( 'paid', 'jetpack' ),
```

Comment:
I wonder if we should add some context here with `_x`, to help translators? Something like "Short label appearing near a block requiring a paid plan."

Reply:
The context would be great. 👍  Otherwise, someone could translate it as "they've already paid for something" vs "this block requires paid plan"